Makefile

```diff
@@ -620,7 +620,7 @@ endef
 
 # $BUILD_TAGS variable is used in hack/golangci-lint.sh
 .PHONY: varlink_generate
-ifneq (or $(findstring varlink,$(BUILDTAGS)),$(findstring varlink,$(BUILD_TAGS)))
+ifneq (,$(or $(findstring varlink,$(BUILDTAGS)),$(findstring varlink,$(BUILD_TAGS))))
```

Comment:
I am still confused about how this is supposed to work for the `make golangci-lint` target. (See the gating failure in https://cirrus-ci.com/task/5884302384693248). It's true that `hack/golangci-lint.sh` sets the `BUILD_TAGS` variable, but that happens after this part here is executed. So it will never have an effect here.

I guess this only ever worked because the condition was broken?